python_modules/dagster-pipes/dagster_pipes/__init__.py

```diff
@@ -88,6 +88,10 @@ class PipesMessage(TypedDict):
     params: Optional[Mapping[str, Any]]
 
 
+PIPES_OPENED_MESSAGE: PipesMessage = _make_message("opened", None)
+PIPES_CLOSED_MESSAGE: PipesMessage = _make_message("closed", None)
```

Comment:
it seems very strange to make this a global. why not just create them on demand?